File: app/src/main/java/com/shuniuyun/material/fragment/TabLayoutFragment2.java
package com.shuniuyun.material.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.shuniuyun.material.R;


public class TabLayoutFragment2 extends Fragment {
    private View view;

    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        view = LayoutInflater.from(getContext()).inflate(R.layout.fragment_tablayout, container, false);
        return view;
    }

    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);
        TextView test = view.findViewById(R.id.test);
        test.setText("第2页");
    }

}

File: app/src/main/java/com/shuniuyun/material/fragment/TabLayoutFragment2.kt
package com.shuniuyun.material.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import com.shuniuyun.material.R

/**
 *  Created by wangyanbin
 */
class TabLayoutFragment2 : Fragment() {
    private var childView: View? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        childView = LayoutInflater.from(context).inflate(R.layout.fragment_tablayout, container, false)
        return childView
    }

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        val test = childView?.findViewById<TextView>(R.id.test)
        test?.text = "第2页"
    }

}